lemur/static/app/angular/certificates/certificate/tracking.tpl.html

```diff
@@ -35,11 +35,11 @@
                ng-maxlength="64"
                ng-blur="certificate.attachCommonName()"
                ng-focus="certificate.removeCommonName()"
+               ng-change="certificate.checkCommonNameOptional(trackingForm)"
                required/>
 
         <p ng-show="trackingForm.commonName.$invalid && !trackingForm.commonName.$pristine" class="help-block">
-          You must
-          enter a common name and it must be less than 64 characters</p>
+          Common name is mandatory unless using Subject Alternate Names and it must be less than 64 characters</p>
```

Comment:
Suggestion:
Either common name or subject alternate name(s) are required, and they must be less than 64 characters